Makefile

```diff
@@ -12,7 +12,11 @@ all: dist
 clean:
 	rm -rf $(build_dir)
 
-dist: install-npm-deps install-bower-deps optimize-js
+dist: install-composer-deps install-npm-deps install-bower-deps optimize-js
```

Comment:
please rename ``dist`` to ``install-deps`` and remove the ``optimize-js`` rule from there